src/drivers/acpi/acpi.zig

```diff
@@ -1,6 +1,6 @@
 const ft = @import("../../ft/ft.zig");
 const tty = @import("../../tty/tty.zig");
-const utils = @import("../../shell/utils.zig");
+const utils = @import("../../shell/default/utils.zig");
```

Comment:
utils is used here for colors in the logger, if the use of these constant has nothing to do with the shell, maybe they should be in another file

Reply:
let's code a lib termcap lmao (I'm joking).

But yeah, I had the same feeling. The fact is I don't really know where I can put it... :thinking:

Reply:
Done 27df2e6

I created a "colors" module, thus we can simply import("colors").